sormas-ui/src/main/java/de/symeda/sormas/ui/utils/AbstractView.java

```diff
@@ -46,6 +46,8 @@ public abstract class AbstractView extends VerticalLayout implements View {
 
 	private static final long serialVersionUID = -1L;
 
+	public static final String DELETE_REASON_COLUMN = "deleteReasonCumulated";
```

Comment:
this constant is not used in this class, should be deleted or moved to the class(es) where it is used

Reply:
This constant is used in the following views:
![image](https://github.com/SORMAS-Foundation/SORMAS-Project/assets/47103965/229b6ad7-c01b-44d8-b7a5-8b4a7e185243)
All those views are extending AbstractView and I wanted to avoid the duplication of the constants in the child views.

Besides the above comment and this one, there is another constant in FilteredGrid (already in the project from the past) should I change that part too?
![image](https://github.com/SORMAS-Foundation/SORMAS-Project/assets/47103965/0377c001-f7c7-4aa8-aea4-a70b98125679)

Reply:
Lets keep the one on the filtered grid, as it is used in almost all grids. 
All the delete reason columns are defined using that constant so that one must be used in the views too.